mephisto/providers/mturk/utils/script_utils.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, TYPE_CHECKING
+
+from mephisto.providers.mturk.mturk_utils import give_worker_qualification
+from mephisto.data_model.requester import Requester
+
+if TYPE_CHECKING:
+    from mephisto.data_model.database import MephistoDB
+
+def direct_soft_block_mturk_workers(
+    db: "MephistoDB",
+    worker_list: List[str],
+    soft_block_qual_name: str,
+    requester_name: Optional[str] = None,
+):
+    """
+    Directly assign the soft blocking MTurk qualification that Mephisto 
```

Comment:
Maybe it's worth noting that `worker_list` here should be a list of MTurk IDs, not a list of worker IDs as listed in the Mephisto database?